easybuild/easyconfigs/p/PyTables/PyTables-3.8.0-foss-2021b.eb

```diff
@@ -0,0 +1,72 @@
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'PyTables'
+version = '3.8.0'
+
+homepage = 'https://github.com/PyTables/PyTables'
+description = """PyTables is a package for managing hierarchical datasets 
+and designed to efficiently cope with extremely large amounts of data."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('Ninja', '1.10.2'),
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('h5py', '3.6.0'),
+    ('scikit-build', '0.11.1'),
+    ('SciPy-bundle', '2021.10'),
+    ('HDF5', '1.12.1'),
+    ('LZO', '2.10'),
+    ('Blosc', '1.21.1'),
+    ('py-cpuinfo', '9.0.0'),
+]
+
+use_pip = True
+download_dep_fail = True
```

Comment:
Should be removed for `PythonBundle` (it trips up the CI as well).

Reply:
```suggestion
```